src/actions/oldInvitationsActions.js

```diff
@@ -55,7 +55,6 @@ export const fetchOldInviteNotificationsAction = (theWalletId?: string = '') =>
     } = getState();
 
     if (accessTokens === undefined || !accessTokens.length) {
-      Sentry.captureMessage('Empty connection access tokens, dispatching restoreAccessTokensAction', { level: 'info' });
```

Comment:
Why do you think we don't need this?

Reply:
this will almost always trigger because accessTokens are not used anymore, the condition is needed for old non-migrated connections yet

Reply:
👍 thanks